x/bank/bench_test.go

```diff
@@ -57,3 +57,33 @@ func BenchmarkOneBankSendTxPerBlock(b *testing.B) {
 		benchmarkApp.Commit()
 	}
 }
+
+func BenchmarkOneBankMultiSendTxPerBlock(b *testing.B) {
+	benchmarkApp, _ := getBenchmarkMockApp()
+
+	// Add an account at genesis
+	acc := &auth.BaseAccount{
+		Address: addr1,
+		// Some value conceivably higher than the benchmarks would ever go
+		Coins: sdk.Coins{sdk.NewInt64Coin("foocoin", 100000000000)},
+	}
+	accs := []auth.Account{acc}
+
+	// Construct genesis state
+	mock.SetGenesis(benchmarkApp, accs)
+	// Precompute all txs
+	txs := mock.GenSequenceOfTxs([]sdk.Msg{multiSendMsg1}, []uint64{0}, []uint64{uint64(0)}, b.N, priv1)
```

Comment:
isn't it possible `b.N` change during runtime?

Reply:
What is `b.N`?  I just copied this directly from what we had earlier, and changed the function name